_redirects

```diff
@@ -1,8 +1,3 @@
 # Place netlify redirects here. https://docs.netlify.com/routing/redirects/
-/regions/*          /
-/vaccination-sites  /
-/who-we-are         /about-us
-/zh/*               /zh-hans/:splat
-# Give our volunteer form a nice url
-/volunteer          https://airtable.com/shrdDabBhmG0l9mO0
-/Volunteer          https://airtable.com/shrdDabBhmG0l9mO0
+/embed https://vaccinatethestates.com/embed 301!
```

Comment:
Rules are read top to bottom. Additionally, Netlify so kindly passes along query params so that should all be handled! https://answers.netlify.com/t/changes-to-redirects-with-query-string-parameters-are-coming/23436